code/modules/projectiles/guns/grenade_launcher.dm

```diff
@@ -36,7 +36,7 @@
 	if(grenades.len)
 		fire_grenade(target,user)
 	else
-		usr << "<span class='danger'>The grenade launcher is empty.</span>"
+		user << "<span class='danger'>The grenade launcher is empty.</span>"
```

Comment:
tiny fix. Also, this grenade launcher is not a real gun, it shouldn't be a child of gun and shouldn't be in the folder.